experimental/python-flask/templates/simple/.appsody-config.yaml

```diff
@@ -0,0 +1 @@
+stack: appsody/python-flask:0.2
```

Comment:
Sorry, it should be 0.1 here rather than 0.2

Reply:
ah, put Chris Bailey had an earlier version of this stack that was 0.1 - so I didn't want to get them confused....

Reply:
0.1.0 was never released or merged so it makes more sense to start from 0.1.0